File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.py
class Solution:
    def getMinDistance(self, nums: List[int], target: int, start: int) -> int:
        min_distance=10001
        for i in range(len(nums)):
            if nums[i]==target and abs(i-start)<min_distance:
                min_distance=abs(i-start)
        return min_distance

File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.c
int valoreassoluto(int number){
    if(number<0) return number*(-1);
    return number;
}

int getMinDistance(int* nums, int numsSize, int target, int start){
    int min_distance=10001;
    int i;
    for(i=0;i<numsSize;i++){
        if(nums[i]==target && (valoreassoluto(i-start))<min_distance) min_distance=valoreassoluto(i-start);
    }
    return min_distance;
}

